Entity/Banner.php

```diff
@@ -0,0 +1,425 @@
+<?php
+
+namespace WebtownPhp\BannerBundle\Entity;
+
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Banner
+ *
+ * @ORM\Table()
+ * @ORM\Entity(repositoryClass="Webtown\EmailBundle\Entity\BannerRepository")
+ */
+class Banner
+{
+    /**
+     * @var integer
+     *
+     * @ORM\Column(name="id", type="integer", nullable=false)
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="name", type="string", length=255, nullable=false)
+     */
+    private $name;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="target", type="string", length=255)
+     */
+    private $target;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="place", type="string", length=255, nullable=false)
+     */
+    private $place;
+
+    /**
+     * @var boolean
+     *
+     * @ORM\Column(name="is_flash_enabled", type="boolean", nullable=false)
+     */
+    private $isFlashEnabled;
+
+    /**
+     * @var boolean
+     *
+     * @ORM\Column(name="is_image_enabled", type="boolean", nullable=false)
+     */
+    private $isImageEnabled;
+
+    /**
+     * @var boolean
+     *
+     * @ORM\Column(name="is_html_enabled", type="boolean", nullable=false)
+     */
+    private $isHtmlEnabled;
+
+    /**
+     * @var integer
+     *
+     * @ORM\Column(name="priority", type="integer", nullable=false)
+     */
+    private $priority;
+
+    /**
+     * @var integer
+     *
+     * @ORM\Column(name="max_display_count", type="integer", nullable=false)
+     */
+    private $maxDisplayCount;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(name="start_at", type="datetime")
+     */
+    private $startAt;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(name="end_at", type="datetime")
+     */
+    private $endAt;
+
+    /**
+     * @var integer
+     *
+     * @ORM\Column(name="display_count", type="integer", nullable=false)
+     */
+    private $displayCount;
+
+    /**
+     * @var integer
+     *
+     * @ORM\Column(name="click_count", type="integer", nullable=false)
+     */
+    private $clickCount;
+
+    /**
+     * @var boolean
+     *
+     * @ORM\Column(name="is_active", type="boolean", nullable=false)
+     */
+    private $isActive;
+
+
```

Comment:
Az áttekinthetőség végett kerüljön ide egy határoló comment:

```php
/**
 * ===========================================================================================
 *                       B E G I N   S E T T E R S   A N D   G E T T E R S
 */
```

És a végére is egy, hogy:

```php
/**
 *                       E N D   S E T T E R S   A N D   G E T T E R S
 * ===========================================================================================
 */
```

Ez nagyban segíti a fejlesztést később, hogy egyértelműen látszik majd, milyen extra (ezen blokkon kívüli) függvényekkel van felvértezve az adott Entity, és hogy hova kell generálni/helyezni az új set/getters-eket.